README.md

```diff
@@ -397,6 +394,132 @@ const { hash } = await erc165SessionKeyProvider.sendUserOperation({
 });
 ```
 
+### Session Key Validator
+
+```ts
+// 1. Get the default ecdsa validator provider
```

Comment:
This is necessary -- not comment

Reply:
Not sure what you mean here.

Reply:
Oh I'm just going over every step.  I'm saying that this step is necessary so we can leave it as is.